files/puppet_helper.groovy

```diff
@@ -170,10 +170,13 @@ class Actions {
       )[0].getStore()
 
     def credentials
+    if (id == "") {
+      id = null
+    }
```

Comment:
> I'm not sure how you would pass null from the cli.

That's pretty much the issue I came up against, which is why I ended up taking this route.